lib/datadog/opentelemetry/trace.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require_relative '../tracing/trace_digest'
+
+module Datadog
+  module OpenTelemetry
+    # OpenTelemetry utilities related to the respective Datadog trace.
+    module Trace
+      class << self
+        # Creates a new TraceOperation object that can be attached to a new
+        # OpenTelemetry span.
+        # If `parent_span` is provided, then that span is set as the currently
+        # active parent span.
+        #
+        # @return [TraceOperation]
+        def start_trace_copy(trace, parent_span: nil)
+          digest = if parent_span
+                     digest_with_parent_span(trace, parent_span)
+                   else
+                     trace.to_digest
+                   end
+
+          # Create a new TraceOperation, attached to the current Datadog Tracer.
+          # This is important because trace flushing is performed by the active
+          # tracer instance.
+          Tracing.send(:tracer).send(:start_trace, continue_from: digest)
```

Comment:
Could you elaborate the difference between this approach and leveraging our public API, such as 
`Datadog::Tracing.continue_trace!(digest)`?

Reply:
It's not needed actually, I think it was required during development, but I changed it back to using ` Datadog::Tracing.continue_trace!(digest)` and it passes tests.